src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -188,6 +192,35 @@
       toggleRouter(br, e, atBookCenter);
     }
 
+    var initialX;
+    var initialY;
+    var holdOffOnToggle = false;
```

Comment:
SUGGESTION: Since `toggleRouter()` above relies on `holdOffOnToggle` being in its closure variables, you might consider moving the `holdOffOnToggle` declaration up above to make it explicit, rather than relying on the reader recognizing the variable hoist that happens from it being declared down here.